From e0955f3ecd1a3d8dbfd3e8affed0077a467665d4 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Sun, 1 Apr 2012 18:37:10 +0800
Subject: [PATCH 1264/2463] ENGR00178646-2 [MX6]Add SD1 and SD2 to interactive
 governor

Different have different SD ports, need to add all SD irqs to
be condition of CPUfreq change and adjust the default irq threshold.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/mach-mx6/irq.c |   12 ++++++++++--
 1 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/irq.c b/arch/arm/mach-mx6/irq.c
index 753767b..2434c1b 100644
--- a/arch/arm/mach-mx6/irq.c
+++ b/arch/arm/mach-mx6/irq.c
@@ -57,12 +57,20 @@ static struct irq_tuner mxc_irq_tuner[] = {
 	 .up_threshold = 0,
 	 .enable = 1,},
 	{
+	 .irq_number = 54, /* uSDHC1 */
+	 .up_threshold = 4,
+	 .enable = 1,},
+	{
+	 .irq_number = 55, /* uSDHC2 */
+	 .up_threshold = 4,
+	 .enable = 1,},
+	{
 	 .irq_number = 56, /* uSDHC3 */
-	 .up_threshold = 8,
+	 .up_threshold = 4,
 	 .enable = 1,},
 	{
 	 .irq_number = 57, /* uSDHC4 */
-	 .up_threshold = 8,
+	 .up_threshold = 4,
 	 .enable = 1,},
 	{
 	 .irq_number = 71, /* SATA */
-- 
1.7.7.4

